docs/pages/choose-an-edition/migrate-to-cloud.mdx

```diff
@@ -0,0 +1,324 @@
+
+1. Determine the `teleport.sh` subdomain you would like to use for your new
+   Teleport Enterprise (cloud-hosted) account. 
+
+1. If your existing Teleport Enterprise (self-hosted) license dashboard is
+   already using your desired subdomain, you can contact Teleport Support to
+   free up the domain for re-use.
+
+   [Reach out](https://goteleport.com/signup/enterprise/) to your Account
+   Management team to set up your Teleport Enterprise (cloud-hosted) tenant.
+
+1. Ensure you are running Teleport Enterprise agents with versions that are
+   lower than the Teleport Enterprise (cloud-hosted) tenant version. To check
+   the versions of your Teleport Enterprise agents, you can use the `tctl`
+   command to list the inventory of connected agents and their version:
+
+   ```code  
+   $ tctl inventory ls --older-than=<version>
+   ```
+
+Validate connectivity to both the newly deployed Teleport Enterprise
+(cloud-hosted) cluster and your existing Teleport Enterprise (self-hosted)
+cluster. You should be able to connect to both Teleport clusters and execute
+`tctl` commands using your current credentials.
+
+1. Log in to the self-hosted Teleport Enterprise cluster:
+
+   ```code
+   $ tsh login --proxy=<Var name="enterprise.example.com" /> --user=<Var name="myuser" />
+   $ tctl status
+   ```
+
+1. Log in to the cloud-hosted Teleport Enterprise cluster:
+
+   ```code
+   $ tsh login --proxy=<Var name="example.teleport.sh" /> --user=<Var name="myuser" />
+   $ tctl status
+   ```
+
+1. Subscribe to the [Teleport Enterprise status
+   website](https://status.teleport.sh/) to stay current on any issues impacting
+   Teleport Enterprise (cloud-hosted) performance.
+
+<Notice
+  type="danger"
+>
+
+  Ensure that the recovery codes displayed when you first set up your Teleport Enterprise (cloud-hosted) tenant are saved securely, so as not to lose access. For your security, Teleport Support cannot assist with resetting passwords or recovering lost credentials.
+
+</Notice>
+
+## Step 2/4. Migrate Teleport resources
+
+After ensuring that both your self-hosted and cloud-hosted Teleport Enterprise
+clusters are up and running, you can migrate dynamic Teleport resources from one
+cluster to the next.
+
+Dynamic Teleport resources such as roles and local users are stored on the
+Teleport Auth Service backend. Since your self-hosted Teleport Enterprise
+cluster uses a separate Auth Service backend from your cloud-hosted cluster, you
+must retrieve the resources on the first backend, then re-apply them against the
+second backend.
+
+Review the [dynamic resources](https://goteleport.com/docs/reference/resources/)
+list to see if any other resources need to be migrated. Some common dynamic
+resources includes:
+
+- `windows_desktop`
+- `apps`
+- `dbs`
+- `login_rule`
+
+<Admonition
+  type="tip"
+  title="Note"
+>
+
+  Trusted clusters are not supported on Teleport Enterprise (cloud-hosted). You
+  will not be able to migrate trusted cluster resources.

```

Comment:
It might be worth mentioning this much closer to the top, so people don't get this far and then realise it won't work for them.